storage/innobase/include/clone0api.h

```diff
@@ -0,0 +1,256 @@
+/*****************************************************************************
+
+Copyright (c) 2017, 2024, Oracle and/or its affiliates.
+
+This program is free software; you can redistribute it and/or modify it under
+the terms of the GNU General Public License, version 2.0, as published by the
+Free Software Foundation.
+
+This program is designed to work with certain software (including
+but not limited to OpenSSL) that is licensed under separate terms,
+as designated in a particular file or component or in included license
+documentation.  The authors of MySQL hereby grant you an additional
+permission to link the program and your derivative works with the
+separately licensed software that they have either included with
+the program or referenced in the documentation.
+
+This program is distributed in the hope that it will be useful, but WITHOUT
+ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+FOR A PARTICULAR PURPOSE. See the GNU General Public License, version 2.0,
+for more details.
+
+You should have received a copy of the GNU General Public License along with
+this program; if not, write to the Free Software Foundation, Inc.,
+51 Franklin St, Fifth Floor, Boston, MA 02110-1301  USA
+
+*****************************************************************************/
+
+/** @file include/clone0api.h
+ Innodb Clone Interface
+
+ *******************************************************/
+
+#ifndef CLONE_API_INCLUDE
+#define CLONE_API_INCLUDE
+
+#include "univ.i"
+#ifndef UNIV_HOTBACKUP
+#include "handler.h"
+#include <fil0fil.h>
+
+using space_id_t = decltype(fil_space_t::id);
+using page_no_t = uint32_t;
```

Comment:
Do we really have to include both headers in this code? The `fil0fil.h` only seems to be needed for this definition of `space_id_t`. Starting with ca501ffb04246dcaa1f1d433d916d8436e30602e we can simply use `uint32_t`.